test/modules/fetch/intercept/fetch.browser.test.ts

```diff
@@ -82,7 +82,7 @@ describe('HTTP', () => {
     expect(request.url.href).toEqual(url)
     expect(request.headers.get('x-custom-header')).toEqual('yes')
     expect(request.credentials).toEqual('same-origin')
-    expect(request.body).toEqual('{"body":"post"}')
+    expect(await request.text()).toEqual('{"body":"post"}')
```

Comment:
This is actually a great opportunity to have some transient assertions: let's read request body as JSON here! Would make the "toEqual" part easier to manage than stringified objects.